src/bundle/Resources/public/css/structure.css

```diff
@@ -0,0 +1,64 @@
+html {
+    height: 100%;
+}
+
+body {
+    min-height: 100%;
+    display: flex;
+}
+
+ez-platform-ui-app {
+    display: flex;
+    flex: 1 0 100%;
+    flex-direction: column;
+}
+
+ez-navigation-hub {
+    flex: 0 0;
+}
+
+main {
+    flex: 1 0;
+    display: flex;
+}
+
+#discovery {
+    flex: 0 0 115px;
+    width: 115px;
+    background: var(--ez-color-discovery-bar);
+    color: var(--ez-color-text-white);
+}
+
+ez-toolbar .ez-button {
```

Comment:
for buttons in discovery/action bar, I think it's just a special case for ez-button, so I think there's no need for `ez-action` and `ez-action-*` classes. Any opinion @inakijv @SylvainGuittard ?

Also, for icons, I think we should now use SVG instead of font icons (see https://cloudfour.com/thinks/seriously-dont-use-icon-fonts/)